x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/connector/syncjob/ConnectorSyncJobIndexServiceTests.java

```diff
@@ -151,6 +154,24 @@ public void testCreateConnectorSyncJob_WithMissingConnectorId_ExpectException()
         );
     }
 
+    public void testDeleteConnectorSyncJob() throws Exception {
+        PostConnectorSyncJobAction.Request syncJobRequest = ConnectorSyncJobTestUtils.getRandomPostConnectorSyncJobActionRequest(
+            connector.getConnectorId()
+        );
+        PostConnectorSyncJobAction.Response response = awaitPutConnectorSyncJob(syncJobRequest);
+        String syncJobId = response.getId();
+
+        assertThat(syncJobId, notNullValue());
+
+        DeleteResponse deleteResponse = awaitDeleteConnectorSyncJob(syncJobId);
+
+        assertThat(deleteResponse.status(), equalTo(RestStatus.OK));
```

Comment:
nit/todo: once the `get` endpoint is implemented, we should try querying for the doc with that id